### DISCUSSION TITLE
Symfony Error on dep init

### DISCUSSION THREAD

**Discussion by Nukro** [deployphp/deployer#2589] [Help needed]
I am getting this error when I try to use "dep init". Does somebody understand why?

![Bildschirmfoto 2021-06-24 um 16 40 19](https://user-images.githubusercontent.com/12048067/123282796-1b0b1600-d50b-11eb-86ff-7e1e6acfc4b3.png)

**Comment by antonmedv** [comment]
Looks like vendors conflict. Use phar version ir deployer/dist
